### ISSUE TITLE
"Align Entries" no longer works for key-values of a section

### ISSUE THREAD

**Issue by chinoto** [tamasfe/taplo#128]
I think the fix for #109 may have broken the "Align Entries" feature for key-values of a section. It works correctly in `0.12.2` and breaks in `0.12.3`.
Result of trying to format with Even Better TOML with the setting enabled in VS Code:
```toml
[dependencies]
palette       = "0.5.0"
strength_reduce = "0.2.3"
rayon         = "1.5.0"
```

**Comment by tamasfe** [comment]
Yes, that is indeed the case, sorry about that. I'm currently reworking the formatting code, tracked in #123.

**Comment by tamasfe** [comment]
This should be fixed in taplo `0.6.1`, and vscode extension version `0.13.1`.
